apps/native/app/src/screens/applications/applications.tsx

```diff
@@ -139,13 +133,8 @@ export const ApplicationsScreen: NavigationFunctionComponent = ({
     }
   }, [applicationsRes])
 
-  // Fix for a bug in react-native-navigation where the large title is not visible on iOS with bottom tabs https://github.com/wix/react-native-navigation/issues/6717
-  if (hiddenContent) {
-    return null
-  }
-
   return (
-    <>
+    <SafeAreaView style={{ flex: 1 }}>
```

Comment:
After the update on Android the view did not fully expand which resulted in bottom tabs indicator being rendered on the middle of the screen. So if you see SafeAreaView being added it is because of that.